controllers/rcj_soccer_referee_supervisor/recorder/recorder.py

```diff
@@ -0,0 +1,41 @@
+from controller import Supervisor
+
+
+class VideoRecordAssistant:
+
+    def __init__(self, supervisor, resolution="720p"):
+        self.supervisor = supervisor
+        self.resolution = resolution
+
+        if not isinstance(self.supervisor, Supervisor):
+            raise TypeError()
```

Comment:
```suggestion
            raise TypeError("Unexpected supervisor instance")
```

Reply:
@Adman we could probably mention here that we expect the `supervisor` to be a descendant of `Supervisor` or something like that

Reply:
Agree. Just wanted to point out that messages will help us resolve the issues more quickly when debugging.